packages/flutter/lib/src/material/icon_theme_data.dart

```diff
@@ -14,8 +21,10 @@ class IconThemeData {
   /// An opacity to apply to both explicit and default icon colors.
```

Comment:
Add something about how if this is driven from an animation, it could be out of range, and that's why you might use clampedOpacity?